storage_broker/src/bin/storage_broker.rs

```diff
@@ -196,8 +196,13 @@ impl SubscriptionKey {
 
     /// Parse from FilterTenantTimelineId
     pub fn from_proto_filter_tenant_timeline_id(
-        f: &FilterTenantTimelineId,
+        opt: Option<&FilterTenantTimelineId>,
```

Comment:
Hm, why we need both Option and 'enabled here'?

Reply:
`Option` comes from `optional FilterTenantTimelineId tenant_timeline_id = 2;`
But AFAIR relying on it is not safe, due to protobuf encoding differences between different libraries. Also protobuf spec says that default for everything is zero values.